engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -142,18 +152,34 @@ public boolean init() {
         return true;
     }
 
+    /**
+     *
+     * @return true if initialized.
+     */
     public boolean isInitialized() {
         return initialized;
     }
 
+    /**
+     *
+     * @return true if the pose of the headset is currently considered reliable.
+     */
     public boolean isHeadTracking() {
         return headIsTracking;
     }
 
+    /**
+     *
+     * @param controllerIndex - 0 for left, 1 for right
+     * @return true if the pose of the controller is currently considered reliable.
+     */
```

Comment:
Same comments as above. And I'd also append ", as an integer." to the controllerIndex line.